<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OngletPlaylist</class>
 <widget class="QWidget" name="OngletPlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ActuListe">
       <property name="text">
        <string>Actualiser Liste Playlists</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AjouterPlaylist">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Ajouter Playlist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="ListePlaylist">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SupprimerPlaylist">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Supprimer Playlist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ordre de préférence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="artaz">
       <property name="text">
        <string>Artistes A -&gt; Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="artza">
       <property name="text">
        <string>Artistes Z -&gt; A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="titreaz">
       <property name="text">
        <string>Titre A -&gt; Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="titreza">
       <property name="text">
        <string>Titre Z -&gt; A</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ActuPlaylist">
       <property name="text">
        <string>Actualiser Playlist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Copier">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Copier Playlist</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="ListeMp3">
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="gridSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
